build-support/ci_lists/integration_remote_whitelist.txt

```diff
@@ -44,7 +73,6 @@ tests/python/pants_test/java:nailgun_integration
 tests/python/pants_test/rules:filedeps_integration
 tests/python/pants_test/targets:unknown_arguments_integration
 tests/python/pants_test/task:goal_options_mixin_integration
-tests/python/pants_test/tasks:bootstrap_jvm_tools_integration
```

Comment:
Frequently times out. Going to wait to handle this in https://github.com/pantsbuild/pants/pull/8278.